packages/oas-to-snippet/src/supportedLanguages.ts

```diff
@@ -1,261 +0,0 @@
-import type { SupportedLanguages } from './types.js';
-
-const supportedLanguages: SupportedLanguages = {
-  c: {
-    highlight: 'text/x-csrc',
-    httpsnippet: {
-      lang: 'c',
-      default: 'libcurl',
-      targets: {
-        libcurl: { name: 'libcurl' },
-      },
-    },
-  },
-  clojure: {
-    highlight: 'clojure',
-    httpsnippet: {
-      lang: 'clojure',
-      default: 'clj_http',
-      targets: {
-        clj_http: { name: 'clj-http' },
-      },
-    },
-  },
-  cplusplus: {
-    highlight: 'text/x-c++src',
-    httpsnippet: {
-      lang: 'c',
-      default: 'libcurl',
-      targets: {
-        libcurl: { name: 'libcurl' },
-      },
-    },
-  },
-  csharp: {
-    highlight: 'text/x-csharp',
-    httpsnippet: {
-      lang: 'csharp',
-      default: 'restsharp',
-      targets: {
-        httpclient: { name: 'HttpClient' },
-        restsharp: {
-          name: 'RestSharp',
-          install: 'dotnet add package RestSharp',
```

Comment:
This `supportedLanguages` config now lives at the `DEFAULT_LANGAUGES` config in `languages.ts` and is largely the same except for two changes:

* `api` is no longer listed here under `node` because that's been offloaded to our `httpsnippet-client-api` plugin.
* `install` has moved directly into `@readme/httpsnippet` client configurations so that information is as close to the snippet generator as possible.